internal/metastore/kv/datacoord/kv_catalog_test.go

```diff
@@ -13,8 +13,9 @@ import (
 
 type MockedTxnKV struct {
 	kv.TxnKV
-	multiSave func(kvs map[string]string) error
-	save      func(key, value string) error
+	multiSave          func(kvs map[string]string) error
+	save               func(key, value string) error
+	multiSaveAndRemove func(saves map[string]string, removals []string) error
```

Comment:
add `removals` in multiSave seems better

Reply:
removed this mock and using mocks from internal/kv